### PR TITLE
Integrate cpvlib

### PR DESCRIPTION
**Pull request by Piranias**
**integrate cpvlib into greco_technologies and pvcompare**:
- see [](https://github.com/isi-ies-group/cpvlib/tree/master/cpvlib)
- create a module apply_cpvlib_StaticHybridSystem.py that creates cpv timeseries from weather data

The following steps were realized, as well (if applies):
- [ x] Use in-line comments to explain your code
- [ x] Write docstrings to your code
- [:x: ] For new functionalities: Explain in readthedocs
- [x ] Write test(s) for your new patch of code
- :x: Update the CHANGELOG.md (let's start this after the first release)
- [ x] Apply black (`black . --exclude docs/`)

*Please mark above checkboxes as following:*
- [ ] Open
- [x] Done

:x: Check not applicable to this PR

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl-institut/mvs_eland/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

**Comment by Piranias** [comment]
a pvlib function is deprecated and needs to be changed in cpvlib before merging is possible. see issue https://github.com/isi-ies-group/cpvlib/issues/5

**Comment by Piranias** [comment]
all functions are tested as before. the coverage stayed the same.

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 git+https://github.com/greco-project/greco_technologies.git@dev
 git+https://github.com/greco-project/cpvtopvlib.git@master
```
Comment (SabineHaas) [inline requirements.txt:2]: Can we remove cpvtopvlib, then?

**Review by SabineHaas — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (Piranias) [reply requirements.txt:2]: yes true! I'll add this before merging

**Pull request merged**